src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -165,11 +165,13 @@ export const Map: React.FC<MapProps> = ({
     [checkForRoomHit, takeSeat]
   );
 
-  const { partygoersBySeat, isSeatTaken } = usePartygoersbySeat({
+  const getUserBySeat = useGetUserByPosition({
     venueId,
-    partygoers,
+    positionedUsers: partygoers,
   });
 
+  const isSeatTaken = () => getUserBySeat !== undefined;
```

Comment:
I don't want to include this into useGetUserByPosition because this `isSeatTaken` is a legacy way used in MapGrid(not used in Auditorium). useMapGrid should be refactored

Reply:
Fair enough.

What's the equivalent/alternative in `Auditorium` (v2)?

Reply:
The logic is different there and you just use getUserBySeat and if there is not any, the seat is empty. You can check the logic in useAuditoriumGrid